tests/test_hypersphere.py

```diff
@@ -400,6 +400,34 @@ def test_mean_and_belongs(self):
         result = self.metric.mean([point_a, point_b, point_c])
         self.assertTrue(self.space.belongs(result))
 
+    def test_diameter(self):
+        n_points = 10
+        points = self.space.random_uniform(n_points)
+        result = self.metric.diameter(points)
+        gs.testing.assert_allclose(result.size, 1)
+
+    def test_closest_neighbor(self):
+        """
+        Check that the closest neighbor is one of neighbors.
+        """
+        n_points = 10
+        points = self.space.random_uniform(n_points)
+        point = points[0, :]
+        neighbors = points[1:, :]
+        index = self.metric.closest_neighbor(point, neighbors)
+        closest_neighbor = points[index, :]
+        result = False
+        for i in range(n_points):
```

Comment:
Is there a numpy builtin function that could do this? Something like `np.isin` (-> `gs.isin`)?

Reply:
It seems that ```np.isin``` works element-wise, whereas here I want to find an entire row. But you are right, I found a way to do it with numpy built-in functions :)